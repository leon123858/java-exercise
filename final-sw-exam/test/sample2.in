student 2001 2002 2003 2004 2005
schoolStrategy Excellent,3 Competent,2 NeedsWork,1 Trash,0
designCriterion A1 criteria1_A1.txt
assignment A1 2001 2002,score2_A1_1.txt 2003,score2_A1_1.txt 2004,score2_A1_2.txt 2005,score2_A1_3.txt
assignment A1 2002 2001,score2_A1_2.txt 2003,score2_A1_2.txt 2004,score2_A1_2.txt 2005,score2_A1_1.txt
assignment A1 2003 2001,score2_A1_3.txt 2004,score2_A1_3.txt 2005,score2_A1_2.txt
averageCriterion A1
calculateScore A1 2001 MeanRankingStrategy
findStrength A1 2001 MeanRankingStrategy
findWeakness A1 2001 MeanRankingStrategy
calculateScore A1 2002 MeanRankingStrategy
findStrength A1 2002 MeanRankingStrategy
findWeakness A1 2003 MeanRankingStrategy
calculateScore A1 2004 MeanRankingStrategy
findStrength A1 2005 MeanRankingStrategy
findStrength A3 2002 MeanRankingStrategy