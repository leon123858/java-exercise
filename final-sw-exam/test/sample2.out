A1-2001 was reviewed by 2002. Level: Excellent Competent Excellent Competent
A1-2001 was reviewed by 2003. Level: Excellent Competent Excellent Competent
A1-2001 was reviewed by 2004. Level: Competent NeedsWork Competent NeedsWork
A1-2001 was reviewed by 2005. Level: Trash Excellent Trash Trash
A1-2002 was reviewed by 2001. Level: Competent NeedsWork Competent NeedsWork
A1-2002 was reviewed by 2003. Level: Competent NeedsWork Competent NeedsWork
A1-2002 was reviewed by 2004. Level: Competent NeedsWork Competent NeedsWork
A1-2002 was reviewed by 2005. Level: Excellent Competent Excellent Competent
A1-2003 was reviewed by 2001. Level: Trash Excellent Trash Trash
A1-2003 was reviewed by 2004. Level: Trash Excellent Trash Trash
A1-2003 was reviewed by 2005. Level: Competent NeedsWork Competent NeedsWork
Assignment: A1, Criterion: Knowledge/Understanding, AvgScore: 1.7
Assignment: A1, Criterion: Thinking/Inquiry, AvgScore: 1.8
Assignment: A1, Criterion: Communication, AvgScore: 1.7
Assignment: A1, Criterion: UseOfVisualAids, AvgScore: 1.1
Assignment: A1, Student: 2001, Score: 1.9
Assignment: A1, Student: 2001, Strength: Knowledge/Understanding Thinking/Inquiry
Assignment: A1, Student: 2001, Weakness: UseOfVisualAids
Assignment: A1, Student: 2002, Score: 1.8
Assignment: A1, Student: 2002, Strength: Knowledge/Understanding Communication
Assignment: A1, Student: 2003, Weakness: UseOfVisualAids
Error
Error
Error